lib/config.js

```diff
@@ -123,5 +123,8 @@ module.exports = rc('license-report', {
 	},
 	httpRetryOptions: {
 		maxAttempts: 5
+	},
+	timeout: {
```

Comment:
I would prefer a name like 'httpTimeoutOptions' to be consistent with 'httpRetryOptions'

Reply:
@BePo65 fixed